src/controllers/groupController.js

```diff
@@ -0,0 +1,40 @@
+import { DEFAULT_PAGE, PAGE_LIMIT } from '../config/global';
+import { GroupModel } from '../models/groupModel';
+import { UserModel } from '../models/userModel';
+import { UserGroupModel } from '../models/userGroupModel';
+import GroupService from '../services/groupService';
+
+const groupService = new GroupService(GroupModel, UserModel, UserGroupModel);
+
+const get = (req, res) => {
```

Comment:
I can see why you chose to follow such naming for the methods in your controllers.
However, it might be confusing - especially, take a look at your routes modules :)
Therefore, consider renaming these methods to illustrate their purpose.